sw/device/lib/dif/dif_plic.c

```diff
@@ -113,6 +113,24 @@ static const plic_peripheral_range_t plic_peripheral_ranges[] = {
             },
 };
 
+/**
+ * Get a number of IE, IP or LE registers (IE00, IE01, ...).
+ *
+ * With more than 32 IRQ sources, there is a multiple of these registers to
+ * accommodate all the bits (1 bit per IRQ source).
+ */
+static size_t plic_num_reg(void) {
+  size_t register_num =
+      PLIC_ID_TO_INDEX(kDifPlicIrqIdLast) / PLIC_ID_TO_INDEX_REG_SIZE;
+
+  // Determine whether there are remaining IRQ sources that have a register.
+  if (PLIC_ID_TO_INDEX(kDifPlicIrqIdLast) % PLIC_ID_TO_INDEX_REG_SIZE) {
```

Comment:
Please can we have an explicit `!= 0` here?

Reply:
Done.

Reply:
Also PLIC_ID_TO_INDEX(kDifPlicIrqIdLast) -> kDifPlicIrqIdLast